sample/lib/ui/pages/profile_page.dart

```diff
@@ -238,8 +225,31 @@ class ProfilePage extends StatelessWidget {
     );
   }
 
-  @override
-  Widget build(BuildContext context) {
+  Widget formWidget(formData) {
+    Widget form;
+    if (_editMode) {
+      var controller = TextEditingController();
+      controller.text = formData;
+      controller.addListener(() {
+        final text = controller.text;
+        controller.value = controller.value.copyWith(text: text);
+        // TODO: Update related field with this data in every change.
+      });
+      form = TextFormField(controller: controller, );
```

Comment:
text'in invalid olmasi durumunda kullaniciya bilgi verdigimiz bir yer var mi? sanki yok gibi?

Reply:
text e validasyon da eklemedik zaten daha. Validasyonlarin uzerinden endpointle birlikte gecmek gerekecektir